cmd/localCI/console.go

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2014,2015,2016 Docker, Inc.
```

Comment:
OK. I can't find it - can you point us to an URL for the one you copied, and we can try to backtrack it to attribute correctly. thx.